### PR TITLE
AT-10: change subtraction method to addition

### PR DESCRIPTION
**Pull request by tairazaq**
customer were complaining that if they click on subtraction it adds the numbers instead. 

see line 10

### CODE REVIEW

File: calculator/src/Main.java [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-        subtract(2,3);
+        additions(2,3);
     }
     public static void calculator(int firstNumber, int secondNumber){
 
         int Answer = firstNumber + secondNumber;
 
-        System.out.print("Answer: " +Answer);
+        System.out.println("Answer: " +Answer);
         //yes bro
```
Comment (SalisAmin) [inline calculator/src/Main.java]: Could you please remove this comment yes bro.

Comment (tairazaq) [reply calculator/src/Main.java]: Okay, Thanks Amin. Thank you for spotting that. i will look into it and get it resolve

Comment (tairazaq) [reply calculator/src/Main.java]: @SalisAmin All issue has been resolved. Please have a look again and get back to me. Thanks

**Review by SalisAmin — APPROVED** [review]
Look good to me 

Many thanks

**Pull request merged**